examples/cluster-advanced/01_configmaps.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster02-user-values
+  namespace: org-multi-project
+data:
+  values: |
+    clusterName: cluster02
+    managementCluster: guppy
+    provider: cloud-director
+
+    baseDomain: capvcd.acme.com
+    kubernetesVersion: v1.22.9+vmware.1
+    clusterDescription: test cluster
+    organization: "multi-project"
+
+    cloudDirector:
+      org: vcd_organisation_name
+      ovdc: vcd_ovdc_name
+      ovdcNetwork: ovdc_network_name
+      site: https://vcd-endpoint.acme.com
+
+    controlPlane:
+      catalog: giantswarm
+      replicas: 3
+      sizingPolicy: m1.large
+      template: ubuntu-2004-kube-v1.22.9
+      diskSizeGB: 35
+      storageProfile: "vsan-ftt-2"
+
+    network:
+      extraOvdcNetworks:
+      - ovdc_network_172-32
+      - ovdc_network_172-64
+      loadBalancer:
+        vipSubnet: 10.10.0.0/27
+      staticRoutes:
+      - destination: 10.20.0.0/16
+        via: 172.32.0.1
+      - destination: 10.30.0.0/16
+        via: 172.32.0.1
+      - destination: 192.168.250.0/24
+        via: 172.64.0.1
+      - destination: 10.88.0.0/16
+        via: 172.64.0.1
+
+    nodeClasses:
+      default:
+        catalog: giantswarm
+        sizingPolicy: m1.large
+        template: ubuntu-2004-kube-v1.22.9
+        storageProfile: "vsan-ftt-1"
+
+    nodePools:
+      worker:
+        class: default
+        replicas: 2
+
+    ntp:
+      servers:
+      - "192.168.250.50"
+      - "192.168.250.51"
+
+    osUsers:
+    - name: "root"
+      sudo: "ALL=(ALL) NOPASSWD:ALL"
+      sshAuthorizedKeys:
+      - "ssh public key"
+    - name: "johndoe"
+      sudo: "ALL=(ALL) NOPASSWD:ALL"
+      sshAuthorizedKeys:
+      - "ssh public key"
```

Comment:
I'd get rid of that. I'm not even sure if this works and we should remove this setting I think.

Reply:
it does work, I think it shows how to add users for debugging